sox.common.js

```diff
@@ -197,6 +198,7 @@
       if (filter) queryParams.push(`filter=${filter}`);
       if (order) queryParams.push(`order=${order}`);
       if (limit) queryParams.push(`pagesize=${limit}`);
+      if (page) queryParams.push(`page=${page}`)
```

Comment:
Tiny nitpick, sorry: can you add a semicolon at the end of this line? (I know it's not necessary but it's the style the rest of SOX uses!)